contents/blog/raising-3m-for-os.md

```diff
@@ -23,44 +23,97 @@ So, what are your options for building a company around the project?
+VC means revenue won't bottleneck your project's growth (usage or features), if you spend the money wisely. That's pretty much the advantage. You will obviously have to build a company around the project so you have a way of making money at least in the long run.
+
+Depending very heavily on the firm, you may get a lot of support with meeting early customers to generate revenue later on, hiring, free VC swag and your strategy. You can always ask the partners and founders from their portfolio to get a sense of what working with them is like.
 
 We felt personally the challenge of managing accelerated growth would be a lot more stimulating, and that has certainly been the case so far!
 
 VC hype begets a bigger team, begets results (hopefully), begets hype:
 
 ![VC Hype Cycle](../images/vc-hype-cycle.jpg)
 
-Especially early, VC is buying into your potential value. This makes sense - if your company has a 1% chance of being worth $1Bn or more, it is rational for people to take that bet if the price is right.
+Especially early, VC is buying into your potential value. This makes sense - if the company that forms around your project has a 1% chance of being worth $1Bn or more, it is rational for people to take that bet if the price is right. Open source companies really can make it big - it's a post for another time, but we believe that open source will eat SaaS's lunch in many product categories.
+
+Simply speaking, it is easier to get into production, and there is less of a sense of vendor lock in, which puts of many developers from trying out a new SaaS product. The end result is that your open source repo could get used at huge companies. They probably have needs your free product can't deliver (uptime, better features, support), which you can make money from, to invest back into the project.
+
+The downside of this VC route is that you risk hype<>reality disonance!
 
-The downside of course is that you risk hype<>reality disonance! If you can't achieve results with the money you raised, whilst increasing your burn rate, you are at the mercy of those funding you if you will survive. Not cool, but you took this route!
+If you can't achieve results with the money you raised, whilst increasing your burn rate, you are at the mercy of those funding you if you will survive. Not cool, but you took this route!
 
-You also have investors, who have given you lots of money. It's important not to fall in love with them for this reason and to forget to prioritize the team and customers, hopefully in that order. Do that, and your investors will be very happy :)
+You also have investors, who have given you lots of money. It's important not to fall in love with them for this reason and to forget to prioritize the team and users then customers, hopefully in that order. Do that, and your investors will be very happy :)
```

Comment:
Not sure this sentence "reason and to forget to prioritize the team and users then customers" makes sense"